app/services/stripe/invoice_handler.rb

```diff
@@ -0,0 +1,36 @@
+class InvoiceHandler
+  def initialize(event)
+    @invoice = event.data.object
+  end
+
+  def create_invoice!
+    begin
+    	Invoice.create!(
+        brand:              get_brand,
+        stripe_invoice_id:  @invoice.id,
+        amount:             @invoice.amount_due,
+        data:               @invoice,
+        paid_at:            Time.now.utc
```

Comment:
Thought this would be useful since we know when the invoices go out, and when it's paid. We can show invoices that haven't been paid yet in the account invoice component.